aiohttp/web_protocol.py

```diff
@@ -94,11 +92,7 @@ def __init__(self, manager, *, loop=None,
                  max_headers=32768,
                  max_field_size=8190,
                  lingering_time=10.0,
-                 max_concurrent_handlers=1,
```

Comment:
Is that an accidental removal?

Reply:
`max_concurrent_handlers` is not removed, see below.
But `**kwargs` is dropped

Reply:
Bah! GitHub UI tricked me.